### PR TITLE
[UWP] Enable mouse scrolling for CollectionView

### LINKED ISSUE #9631: [UWP] How to scroll in CollectionView ?
Hello, I would like to know if it is possible to scroll in a collectionView on UWP without using the ScrollBar? That is to say with a Click and slide or with the mouse wheel?

### PR DESCRIPTION
**Pull request by EmilAlipiev**
### Description of Change ###

Enables mouse scrolling for CollectionView on UWP

### Issues Resolved ### 

- closes #11372
- fixes #9631
  
Changed:
 - StructuredItemsViewRenderer.cs 
 
 None

### Platforms Affected ### 
UWP
 
### Testing Procedure ###
Issue9631.cs test case

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -27,34 +27,6 @@ resources:
       name: xamarin/yaml-templates
```
Comment (jsuarezruiz) [inline azure-pipelines.yml:27]: These changes are not necessary, right?. Could you revert the changes?.

Comment (EmilAlipiev) [reply azure-pipelines.yml:27]: well, i didnt change any of these. originally my base is main branch. see my other PR on main. @samhouts asked me to base on 4.8.0 and i dont know why but more changes are marked. It is probably because main has different yaml file than 4.8.0 branch.

Comment (EmilAlipiev) [reply azure-pipelines.yml:27]: sorry this one went wrong. it was supposed to be from issue branch like this one https://github.com/xamarin/Xamarin.Forms/pull/11372/files
Can we delete this PR and concentrate on 11372?
